concepts/reduce/.meta/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "blurb": "Reducing is the repeated application of a function to an sequence and accumulating the results",
+  "authors": [
+    "glennj"
+  ],
+  "contributors": [],
+  "forked_from": ["common-lisp/reducing"]
```

Comment:
Note, Python has a `functools.reduce()` and I think Bethany had a todo item to add a concept to Python on that topic, so this may be reusable there in the future.

Reply:
I think I'll remove this. I didn't actually take anything from that track.